scripts/gdb/lvglgdb/cmds/drivers/nuttx/nuttx.py

```diff
@@ -0,0 +1,58 @@
+# GDB script to get lvgl global pointer in NuttX.
+
+import argparse
+import sys
+
+import gdb
+
+# Add current script folder so we can import module lvgl
+
+
+class Lvglobal(gdb.Command):
+    """Set which lvgl instance to inspect by finding lv_global pointer in task TLS data."""
+
+    def __init__(self):
+        super(Lvglobal, self).__init__("lvglobal", gdb.COMMAND_USER)
+
+    def set_lvgl_instance(self, inst):
+        from lvglgdb.lvgl import curr_inst
+
+        curr_inst().reset()
+        curr_inst().ensure_init(inst)
+
+    def invoke(self, arg, from_tty):
+        parser = argparse.ArgumentParser(description=self.__doc__)
+        parser.add_argument("-p", "--pid", type=int, help="Optional process ID")
+        try:
+            args = parser.parse_args(gdb.string_to_argv(arg))
+        except SystemExit:
+            return
+        try:
+            from nxgdb import utils
+        except ImportError:
+            gdb.write("nxgdb is not installed, can't find lvgl global pointer.\n")
```

Comment:
Should we use directly, for simplicity.
```suggestion
            print("nxgdb is not installed, can't find lvgl global pointer.")
```